.mend/scheduler-runs/scout-report-2026-04-06.md

```
@@ -0,0 +1,170 @@
+## 4. Broken/Stalled Plans
+
+| Issue | Plan File | Status | Problem |
+|-------|-----------|--------|---------|
+| #102 | `.mend/plans/ISSUE-102.md` | 📝 Plan Draft | ADR-008 creation still pending |
+| #118 | `.mend/plans/taxonomy-loader.md` reference | plan-needs-work | Stalled - sensor.report.v1 wrapping incomplete |
+
+---
+
+## 5. Other Gaps
+
+### Schema Drift
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)